### PR TITLE
Add no remediation tag to test in sshd_use_approved_kex_ordered_stig

### PR DESCRIPTION
**Pull request by jan-cerny**
The rule `sshd_use_approved_kex_ordered_stig` doesn't have a remediation on RHEL 8 and OL 8. We need to mark the test scenarios of this rule with "remediation = none" tag. This issue has been discovered by review of `/per-rule` test results.

### CODE REVIEW

File: linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/tests/comment.fail.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 # platform = multi_platform_ol,multi_platform_rhel,multi_platform_sle,multi_platform_slmicro,multi_platform_ubuntu,multi_platform_almalinux
+{{% if product in ['ol8', 'rhel8']  %}}
```
Comment (Mab879) [inline linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/tests/comment.fail.sh:3]: We should remove this extra blank line. This applies to all of the files in this PR.

**Review by Mab879 — CHANGES_REQUESTED** [review]

**Comment by jan-cerny** [comment]
I have chnged it to remove the empty line

**Review by Mab879 — APPROVED** [review]
